File: R/aula3.R
#variáveis
nome <- "Marcos"
idade <- 19
altura <- 1.72
brasileiro <- TRUE

print(nome)
print(idade)
print(altura)
print(brasileiro)

print(typeof(nome))
print(typeof(idade))
print(typeof(altura))
print(typeof(brasileiro))

#mensagem
mensagem <- "Qualquer coisa"
print(mensagem)
print(type(mensagem))

#data/hora
data_atual <- Sys.time()
data <- Sys.Date()
hora <- Sys.time()
local <- Sys.timezone()

print(data_atual)
print(data)
print(hora)
print(local)

#enumerar
DiasDaSemana <- factor(c("Segunda", "Terça", "Quarta", "Quinta", "Sexta", "Sabado", "Domingo"), levels = c("Segunda", "Terça", "Quarta", "Quinta", "Sexta", "Sabado", "Domingo"))

print(DiasDaSemana[3])
TRUE | FALSE

File: Python/aula3.py
#variáveis
nome = "Marcos"
idade = 19
altura = 1.72
brasileiro = True

print("Nome: ", nome)
print("Idade: ", idade)
print("Altura: ", altura)
print("Brasileiro: ", brasileiro)

print(type(nome))
print(type(idade))
print(type(altura))
print(type(brasileiro))

#float
PI = 3.14159

#mensagem
mensagem = "qualé rapaziada"
print(mensagem)
print(type(mensagem))

#inteiro
numero = 100

#data/hora
from datetime import datetime
data_atual = datetime.now()
print(data_atual)

#enumerar
from enum import Enum

class DiasDaSemana(Enum):
  Segunda = 1
  Terca = 2
  Quarta = 3
  Quinta = 4
  Sexta = 5
  Sabado = 6
  Domingo = 7

print(DiasDaSemana.Quarta.name)
print(DiasDaSemana.Quarta.value)
